brewtils/plugin.py

```diff
@@ -531,13 +533,14 @@ def _initialize_processors(self):
         # values specified at plugin creation
         connection_info = self._instance.queue_info["connection"]
         if "ssl" in connection_info:
-            connection_info["ssl"].update(
```

Comment:
The changes here ensure that the values supplied by beergarden (the values in `connection_info`) get used if they haven't been explicitly set in the plugin config, as in the case of local plugins.